### DISCUSSION TITLE
Not working Mobile-detect plugin

### DISCUSSION THREAD

**Discussion by m9Is** [YOURLS/YOURLS#3553] [Q&A] [answered]
Hello, the Mobile-detect plugin does not work when you enable it and create a link by type
https://www.youtube.com/watch?v=ghYXdarPiJYiosvnd.youtube://www.youtube.com/watch?v=ghYXdarPiJYandroidosintent://www.youtube.com/watch?v=ghYXdarPiJY#Intent;package=com.google.android.youtube;scheme=https;end
Automatic transition to YouTube applications does not occur
if you turn on yourls toolbar, then when you click on the pages of the close, you go to the desired application, but I would like this to happen automatically without pressing a button on the transition page. 
help me please.

**Comment by dgw** [comment, marked as answer]
You've already asked for help in the most likely place to get it, i.e. the plugin's repository (guessi/yourls-mobile-detect#3).

Based on a glance at the plugin's readme, though, one big URL with special prefixes embedded in it doesn't look like the intended usage. The readme indicates you're meant to create three separate keywords with the relevant URLs: `shrt`, `iosshrt`, and `androidosshrt`.

**Reply by m9Is** [reply]
Unfortunately yourls does not accept such a link [intent://www.youtube.com/watch?v=ghYXdarPiJY#Intent;package=com.google.android.youtube;scheme=https;end] and will not let you create a separate

**Reply by m9Is** [reply]
![Screenshot 2023-04-01 051836](https://user-images.githubusercontent.com/129141659/229242778-b53d2533-ed51-45e1-a43b-8c1ca59e36a9.png)

**Reply by dgw** [reply]
https://yourls.org/docs/guide/advanced/custom-protocols

**Reply by m9Is** [reply]
Working, thanks you.
